url-shortener/src/test/java/com/github/bibsysdev/urlshortener/service/model/UriMapTest.java

```diff
@@ -0,0 +1,24 @@
+package com.github.bibsysdev.urlshortener.service.model;
+
+import static no.unit.nva.testutils.RandomDataGenerator.randomInstant;
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import org.junit.jupiter.api.Test;
+
+public class UriMapTest {
+
+    private static final String DOMAIN = "https://api.sandbox.nva.aws.unit.no";
+    private static final String ID_NAMESPACE = "https://api.sandbox.nva.aws.unit.no/download/short";
```

Comment:
Being a short URI is relative.  :P
The long version is 2000 characters long, so in comparison this is short.